cardshifter-server/src/main/java/com/cardshifter/server/model/MainServer.java

```diff
@@ -71,6 +71,11 @@ public MainServer(ServerConfiguration serverConfiguration) {
 	public Server start() {
         mods.loadExternal(mods.getDefaultModLocation());
         mods.loadExternal(Paths.get(config.getModsDirectory()));
+
+        if(mods.getAvailableMods().isEmpty()) {
+        	logger.warn("No mods were loaded from the different folders. Please check your installation.");
+        	throw new IllegalStateException("No mod loaded");
```

Comment:
It's an illegal state that can happen if the user hasn't put any mods at the right place, it's not a programming error. I would associate illegal state more with a programming error.